dotcom-rendering/src/components/ShareButton.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import { css } from '@emotion/react';
+import { ArticleDesign, ArticleDisplay, Pillar } from '@guardian/libs';
+import type { StoryProps } from '../../.storybook/decorators/splitThemeDecorator';
+import {
+	defaultFormats,
+	splitTheme,
+} from '../../.storybook/decorators/splitThemeDecorator';
+import { palette as themePalette } from '../palette';
+import {
+	CopyLinkButton,
+	NativeShareButton,
+	ShareButton,
+} from './ShareButton.importable';
+
+export default {
+	component: ShareButton,
+	title: 'Components/ShareButton',
+};
+
+interface StoryArgs extends StoryProps {
+	theme: string;
+}
+
+export const CopyLink = () => {
+	return (
+		<ShareButton
+			pageId={'123'}
+			webTitle={'The the'}
+			format={{
+				display: ArticleDisplay.Standard,
+				theme: Pillar.News,
+				design: ArticleDesign.Standard,
+			}}
+			context="ArticleMeta"
+		/>
+	);
+};
+CopyLink.storyName = 'CopyLink';
```

Comment:
I think this is the full share button rather than the copy link one? If you open this story in a browser that supports the share API it will show the native share button, not copy link.

```suggestion
export const ShareButtonStory = () => {
	return (
		<ShareButton
			pageId={'123'}
			webTitle={'The the'}
			format={{
				display: ArticleDisplay.Standard,
				theme: Pillar.News,
				design: ArticleDesign.Standard,
			}}
			context="ArticleMeta"
		/>
	);
};
CopyLink.storyName = 'ShareButton';
```